apps/server/src/modules/learnroom/uc/lesson-copy.uc.spec.ts

```diff
@@ -60,6 +65,7 @@ describe('lesson copy uc', () => {
 		authorisation = module.get(AuthorizationService);
 		courseRepo = module.get(CourseRepo);
 		lessonCopyService = module.get(LessonCopyService);
+		etherpadService = module.get(EtherpadService);
```

Comment:
if `etherpadService` is not used, it is enough to implement it as provider on lines 55-58 without lines 21 and 68

Reply:
I did. However, we can test only the url for example, which should be ok. We can't test the whole object, because the title is a random string, making it difficult to test